packages/notifi-react-card/lib/components/NotifiEmailInput.tsx

```diff
@@ -47,9 +55,22 @@ export const NotifiEmailInput: React.FC<NotifiEmailInputProps> = ({
 
   return (
     <>
-      <label className={clsx('NotifiEmailInput__label', classNames?.label)}>
-        {copy?.label}
-      </label>
+      {intercomView ? (
+        intercomCardView.state === 'settingView' ? (
+          <div
```

Comment:
Nit: This Resend verification link will appear if in the data, the destination has not been verified. You can take steps to start implementing the logic by passing in a isVerified data and/or setting a setState.